data/examples/declaration/warning/warning-single-line-out.hs

```diff
@@ -0,0 +1,4 @@
+{-# WARNING test "This is a warning" #-}
```

Comment:
You transform singleton list into a string? Is this the concern of a formatter?

Reply:
It appears that the inner representation of the two is the same.